### ISSUE TITLE
Discovery for accessibility annotation guidelines (requirements?) at Midpoint Review

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#52453]
## User Story
_"As a VFS team member, I want to get the most complete accessibility feedback I can get at Midpoint Review, so that we can avoid potential launch-blocking issues at our Staging Review."_

_"As the Governance team accessibility specialist, I want to know what decisions have already been made about accessible interactions when designing the Midpoint Review prototype, so I can give useful feedback before the prototype is handed off to the VFS team's engineers."_

Assignee:
Peer Reviewer:

## Description
For accessibility, there's potentially a big gap that happens between a VFS team's Midpoint Review and their Staging Review. At Midpoint, we should be seeing the team's finalized prototype (pending usability testing) -- typically a sequence of static images showing the intended visual appearance of each page of the product under different conditions. At Staging, we have real code that will be navigated and parsed by disabled users and assistive technology. There are a lot of potential interactions that may not be accounted for in the static prototype images.

Whenever possible, the accessibility feedback we give at Midpoint tries to anticipate how engineers may plan to implement the prototype and make code suggestions for different interaction patterns. But when providing that feedback, we often don't know how much the designer has already been thinking about those interactions, or whether the recommendations made at Midpoint will get documented and passed to the engineering team as intended.

One technique that may help is **accessibility bluelining**, or annotating prototypes specifically for accessibility information (example discussions of bluelines: [How accessibility bluelines shaped Adobe Fresco](https://adobe.design/stories/process/how-accessibility-bluelines-shaped-fresco) and [Bluelines: documenting accessible designs, HCDE-ALB video](https://www.youtube.com/watch?v=OhrIpeyGxYQ)). Including bluelined prototypes at Midpoint would:
- Prompt VFS designers to consider types of interactions that may not be on their radar otherwise.
- Show us what kind of accessibility thinking has already gone into the work we're reviewing, so we don't recommend something that's already planned or accounted for.
- Make it easier to provide actionable accessibility recommendations at Midpoint Reviews (eg. "Right now you have keyboard focus moving from X to Y. Consider moving it from X to Z instead, so that...").

The Design System team has already built an annotations library in Sketch. But only a handful of VFS designers are currently doing this. Asking (or requiring?) VFS teams to include accessibility annotations as part of their submitted artifacts would be a big change.

Discovery work completed as part of this ticket should:
- Identify and quantify issues found at Staging Review that could have been prevented by better accessibility feedback at Midpoint Review.
   - Determine if that Midpoint Review feedback would have been easier to provide if the VFS team had provided blueline annotations with their artifacts.
- Determine whether Platform can create clear accessibility annotation guidelines that are usable for VFS teams, and broadly determine what should be included (details to be left to a follow-up ticket).
   - Identify any gaps in the existing Design System annotations library.
- Determine what kind of additional training would be required for VFS teams (especially, but not exclusively, designers) to successfully integrate accessibility annotations into our processes.
   - This should include research with actual VFS designers.
- Make a recommendation for whether accessibility annotations should be required as part of Midpoint Review, or just be optional/highly encouraged.

Useful resources:
- [VA Design System Annotations Library](https://www.sketch.com/s/aaa5c25f-6991-4aac-a6ed-d378bdff7727), which already includes several accessibility-specific symbols
- [Expectations for wireframe accessibility annotations](https://kb.iu.edu/d/bhpk) from Indiana University

This came up in the discussion around #52170, but we decided it was potentially a big enough change that it should be separated out as its own future initiative.

## Tasks
- [ ] _Placeholder task_

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] _Placeholder AC_

### Refinement
- [ ] Ticket has user story, description, tasks, and acceptance criteria
- [ ] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [ ] Ticket has been sized 
- [ ] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [ ] Ticket has been discussed as a team at Planning
- [ ] Ticket has been assigned to appropriate initiative or quarterly epic
- [ ] Ticket has been assigned to a Sprint
- [ ] Ticket has been moved to the Planned pipeline in the Governance team board
- [ ] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
hey @briandeconinck . I discussed requiring accessibility annotations at MPR with Matt at our last 1:1. You had previously shared that Martha would like to make them required at Midpoint Review. Here are my notes from our discussion:

- Matt would like more information on what the value-add would be for requiring accessibility annotations.
- DS needs better guidance to VFS teams re: a11y - how would moving forward with accessibility annotations impact DS components or DST process? or would it?
- Can Figma be a resource here?
- Could this result in less findings at SR? If yes, worth it. 
- Would CAIA a11y also find this helpful?

Could you take a look at this original ticket and make sure it is up to date (granted it's a year old - I'm not sure if anything is no longer relevant)? Would you also see if you can answer Matt's questions whether within the ticket or as a reply to this comment. Please lmk if you have any questions - thank you!

**Comment by shiragoodman** [comment]
this ticket is no longer needed.
